.template/.rubocop.yml

```diff
@@ -0,0 +1,53 @@
+AllCops:
+  NewCops: enable
+  TargetRubyVersion: 2.7
```

Comment:
shouldn't we target ruby 3?

Reply:
Updated in [25f0dc4](https://github.com/nimblehq/rails-templates/pull/320/commits/25f0dc400a0c1130ecb620c0eb6db4c839f746b6) ✨